modules/coingate/coingate.php

```diff
@@ -210,19 +209,12 @@ private function postValidation()
             }
 
             if (empty($this->postErrors)) {
-                $cgConfig = array(
-                    'auth_token' => $this->stripString(Tools::getValue('COINGATE_API_AUTH_TOKEN')),
-                    'environment' => (int)(Tools::getValue('COINGATE_TEST')) == 1 ? 'sandbox' : 'live',
-                    'user_agent' => 'CoinGate - Prestashop v' . _PS_VERSION_
-                        . ' Extension v' . COINGATE_PRESTASHOP_EXTENSION_VERSION,
-                );
-
-                \CoinGate\CoinGate::config($cgConfig);
-
-                $test = \CoinGate\CoinGate::testConnection();
+                $auth_token = $this->stripString(Tools::getValue('COINGATE_API_AUTH_TOKEN'));
```

Comment:
please use https://github.com/coingate/coingate-php#attention-plugin-developers to pass current plugin version